hardhat/tasks/func-mintBridgedToken.js

```diff
@@ -0,0 +1,52 @@
+const { readLastDeployedAddress } = require("../helpers/fileHelpers");
+const { display } = require("../helpers/loggingHelper");
+
+/**
+ * @dev caller MUST BE the admin / deployer
+ */
+task("func-mintBridgedToken", "mint amount of bridged token to user")
```

Comment:
```suggestion
task("func-mintBridgedToken", "token mint amount bridged to user")
```

Reply:
Sorry, I don't understand. Need an explanation.

Suggestion: `mint amount of BridgedToken to user`